src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityFeed-test.js

```diff
@@ -207,6 +199,30 @@ describe('elements/content-sidebar/ActivityFeed/activity-feed/ActivityFeed', ()
         expect(instance.feedContainer.scrollTop).toEqual(100);
     });
 
+    test('should scroll to active feed item', () => {
+        const wrapper = shallow(
+            <ActivityFeed
+                currentUser={currentUser}
+                feedItems={[{ type: 'comment' }]}
+                activeFeedEntryId={comments.entries[0].id}
+            />,
+        );
+        const instance = wrapper.instance();
+        const li = document.createElement('li');
```

Comment:
I suppose we could suggest `mount`ing an `<li/>` instead of calling document directly, but i don't think this matters much in practice.